src/objects/Application__c.object

```diff
@@ -0,0 +1,268 @@
+                    <default>false</default>
+                    <label>Received</label>
+                </value>
+                <value>
+                    <fullName>In Review</fullName>
+                    <default>false</default>
+                    <label>In Review</label>
+                </value>
+                <value>
+                    <fullName>Waitlist</fullName>
+                    <default>false</default>
+                    <label>Waitlist</label>
+                </value>
+                <value>
+                    <fullName>Admit</fullName>
+                    <default>false</default>
+                    <label>Admit</label>
+                </value>
+                <value>
+                    <fullName>Deny</fullName>
+                    <default>false</default>
+                    <label>Deny</label>
+                </value>
+                <value>
+                    <fullName>Deferred Offer</fullName>
+                    <default>false</default>
+                    <label>Deferred Offer</label>
+                </value>
+                <value>
+                    <fullName>Accepted Offer</fullName>
+                    <default>false</default>
+                    <label>Accepted Offer</label>
+                </value>
+                <value>
+                    <fullName>Declined Offer</fullName>
+                    <default>false</default>
+                    <label>Declined Offer</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Application_Type__c</fullName>
+        <description>The type of program or track the applicant is applying to. For example, Undergraduate, Graduate, or Transfer.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The type of program or track the applicant is applying to. For example, Undergraduate, Graduate, or Transfer.</inlineHelpText>
+        <label>Application Type</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Undergrad</fullName>
+                    <default>false</default>
+                    <label>Undergrad</label>
+                </value>
+                <value>
+                    <fullName>Transfer</fullName>
+                    <default>false</default>
+                    <label>Transfer</label>
+                </value>
+                <value>
+                    <fullName>Graduate</fullName>
+                    <default>false</default>
+                    <label>Graduate</label>
+                </value>
+                <value>
+                    <fullName>Post-Doc</fullName>
```

Comment:
I have questions around whether or not this is the correct term (https://en.wikipedia.org/wiki/Postdoctoral_researcher) and if we need to add any additional picklist values here - I'll add my comment to the spec on Monday and followup next week.

Reply:
I wanna say I saw that actually come up in the comments. I personally *hate* calling it Post-Doc and think we should be explicit here, but we should tag Samara in that doc, ultimately it would be her call I think.

Reply:
@xinnanlisf @robertbwatson I think there may be some confusion here as the comments in the spec were a bit mixed up.
The Application Status, should be in that chronological order like it is in the spec doc: Submitted, Incomplete, Received, In Review, Waitlist, Admit, Deny, Deferred Offer, Accepted Offer, Declined Offer 

The Application Type picklist should be in alphabetical order.